tests/verify_riak_object_reformat.erl

```diff
@@ -34,6 +34,11 @@ confirm() ->
     rt:update_app_config(all, [{riak_kv, [{object_format, v1}]}]),
     TestMetaData = riak_test_runner:metadata(),
     DowngradeVsn = proplists:get_value(upgrade_version, TestMetaData, previous),
+
+    %% Use previous version's riak.conf so that when we
+    %% downgrade we don't crash on unknown config keys:
+    rtdev:copy_conf(?N, previous, current),
```

Comment:
This should be routed through rt, not called directly.

(Another alternative would be to use the upgrade callback mechanism, but this is fine, too)